travis/test_server.py

```diff
@@ -252,13 +261,15 @@ def setup_server(db, odoo_unittest, tested_addons, server_path, script_name,
                                 stderr=subprocess.STDOUT,
                                 stdout=subprocess.PIPE)
         for line in iter(pipe.stdout.readline, ''):
+            if line == b'':
+                break
             if hidden_line(line, main_modules=[], addons_path_list=[],
                            hidden_all_no_translation=True):
                 continue
             # No show a warning from template runtime
             # if 'openerp.models.schema' in line:
             #     line = line.replace('DEBUG', 'WARNING')
-            print(line.strip())
+            print(print_log(line))
```

Comment:
@JesusZapata 
I found how to avoid this method.

Using the following patches:
```patch
- for line in iter(pipe.stdout.readline, ''):
+ for line in pipe.stdout:
...
- if line == b'':
-    break
...
-print(print_log(line))
+print(line.strip().decode('UTF-8'))
```

Could you create another PR cleaning this code, please?

Reply:
Wait I have a overlapped issue related.